pkg/config/config_template.yaml

```diff
@@ -1354,6 +1354,11 @@ api_key:
   ##        @param DD_APM_OBFUSCATION_MEMCACHED_ENABLED - boolean - optional
   ##        Enables obfuscation rules for spans of type "memcached". Enabled by default.
   #         enabled: false
+  ##        @param DD_APM_OBFUSCATION_MEMCACHED_KEEP_COMMAND - boolean - optional
```

Comment:
nit: could use a newline between the two memcached options but not a blocker

Reply:
CI has been so difficult to get passing today that I'm going to go ahead and just merge the PR as-is, since it's not a blocker. I appreciate the comment regardless though.